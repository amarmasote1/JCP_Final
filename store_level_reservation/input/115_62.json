{
	"key": "JCP|1786|15345672117|EACH"
}  