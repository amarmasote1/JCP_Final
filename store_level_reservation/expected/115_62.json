{
	"status": 200,
	"message": "Success",
	"key": "JCP|1786|15345672117|EACH",
	"body": {
		"shipSafetyFactor": 0,
		"bopusSafetyFactor": 0,
		"shipSafetyFactorSource": 0,
		"pickSafetyFactorSource": 0,

		
		
		
		"reservedDemand": 0,
		"unconfirmedDemand": 0,
		
		"dirty": false
	},
	"warnings": null
}